README.md

```diff
@@ -129,6 +129,33 @@ env "gorm" {
 }
 ```
 
+### Supported Features
+#### View
+To create a view, you can define the `ViewDef` method on a struct:
+
+```go
+type User struct {
+  gorm.Model
+  Name string
+  Age  int
+}
+
+type WorkingAgedUsers struct {}
+
+func (WorkingAgedUsers) ViewDef(db *gorm.DB) gorm.ViewOption {
```

Comment:
```suggestion
func (WorkingAgedUsers) ViewDef(db *gorm.DB) gormschema.ViewDef {
```

(Our) ViewDef should be something like https://github.com/ariga/atlas/blob/master/sql/schema/schema.go#L49-L58. Raw definition and maybe additional attributes we'll use in the future to extend this (e.g., Deps).

Reply:
Yes, for two reasons:
1. triggers, views and future additions to this should have the ~same API. They give the user full flexibility and expect the same output - raw definitions. 
2. The gorm builder does not support all options of all databases. If the underlying result is raw definition, users can always fallback to it and define the `CREATE` statement themselves.